presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -803,6 +805,22 @@ protected Type visitFunctionCall(FunctionCall node, StackableAstVisitorContext<C
                 throw e;
             }
 
+            if (node.getSortItems().isPresent()) {
```

Comment:
> com.facebook.presto.sql.analyzer.SemanticException: line 1:97: Column 'c' cannot be resolved
Why `c` isn't resolved? Are you using legacy `ORDER BY` mode: `deprecated.legacy-order-by`?

Does this work:
```
presto:tiny> explain select 1 as c, nationkey from customer group by nationkey order by sum(c order by custkey)
```